src/core/packages/saved-objects/migration-server-internal/src/common/utils/logs.ts

```diff
@@ -47,18 +48,14 @@ export const logStateTransition = (
   }
 
   const logMessage = `${logPrefix}${prevState.controlState} -> ${currState.controlState}. took: ${tookMs}ms.`;
-  if (logger.isLevelEnabled('debug')) {
-    logger.debug<StateTransitionLogMeta>(logMessage, {
-      kibana: {
-        migrations: {
-          state: currState,
-          duration: tookMs,
-        },
+  logger.debug<StateTransitionLogMeta>(logMessage, {
```

Comment:
Only log state transitions at the debug level... This should reduce our verbosity quite a bit.